trimmer_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length >= 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```

Comment:
- [ ] The length of the name in the condition should be greater than 10, not greater or equal to 10. Please remove the symbol `=`. Remember, if the name length is longer than 10 then it should trim the word.

Reply:
All good now 👍🏽